CMakeLists.txt

```diff
@@ -244,6 +244,11 @@ if(BUILD_TESTS)
     NAME member_client_test
     COMMAND ${PYTHON} ${CMAKE_SOURCE_DIR}/tests/memberclient.py -b . -p libloggingenc ${CCF_NETWORK_TEST_ARGS})
 
+  ## Add node test
+  add_test(
```

Comment:
It looks like the base commit used to diff this PR is from an older version of `master`. We should use the new `add_e2e_test` wrapper here.

Reply:
Thanks, I'll change it